CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 * Your contribution here.
 
+* [#1740](https://github.com/ruby-grape/grape/pull/1740): Fix dependent parameter validation using `given` when parameter is a `Hash` - [@jvortmann](https://github.com/jvortmann).
```

Comment:
Nitpick, put "Your contribution here" below the line? Just more consistent with how we've been doing it.